### ISSUE TITLE
Update MultiMC

### ISSUE THREAD

**Issue by leo60228** [NixOS/nixpkgs#35189]
## Issue description
MultiMC in master is on version 0.5.1. The latest is 0.6.1, which has several new features.

### Steps to reproduce
Look at <https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/multimc/default.nix>.

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.
I'm on the unstable channel, but I was just looking at the repo, so I don't think this is relevant.

**Comment by NeQuissimus** [comment]
I tried updating it briefly but there is something wrong with the C build that I cannot figure out:
It has something to do with the fact that quazip needs to be updated.
Maybe somebody can give this a shot :)

```
[ 18%] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/src/LockedFile_unix.cpp.o
/tmp/nix-build-multimc-0.6.1.drv-0/source/libraries/classparser/src/classparser.cpp:21:24: fatal error: quazipfile.h: No such file or directory
 #include <quazipfile.h>
                        ^
compilation terminated.
make[2]: *** [libraries/classparser/CMakeFiles/MultiMC_classparser.dir/build.make:63: libraries/classparser/CMakeFiles/MultiMC_classparser.dir/src/classparser.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1274: libraries/classparser/CMakeFiles/MultiMC_classparser.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 19%] Building CXX object libraries/LocalPeer/CMakeFiles/LocalPeer.dir/LocalPeer_autogen/mocs_compilation.cpp.o
```

**Comment by leo60228** [comment]
I had the same problem when I tried.

**Comment by dtzWill** [comment]
I'm working on this now, should have something in a bit....

**Comment by dtzWill** [comment]
Should be fixed in #35221, but I'd appreciate testing from someone familiar with MultiMC as confirmation. Please! :)
